### PR TITLE
bump to 24.0.7

### PR DESCRIPTION
**Pull request by blizzz**
* [x] test fresh install
* [x] test upgrade
* [x] request publish
* [x] assert published

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
🐘

File: resources/entrypoint.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 if [ -f "/var/www/html/occ" ]; then
     # Apply one-click-instance settings...
     sudo -u www-data php /var/www/html/occ config:system:set one-click-instance --value=true --type=bool
-    sudo -u www-data php /var/www/html/occ config:system:set one-click-instance.user-limit --value=100 --type=int
+    sudo -u www-data php /var/www/html/occ config:system:set one-click-instance.user-limit --value=500 --type=int
```
Comment (szaimen) [inline resources/entrypoint.sh:5]: Where was this requested? Should we do the same on AIO?

**Review by szaimen — CHANGES_REQUESTED** [review]

File: resources/entrypoint.sh (hunk 1 above)
Comment (blizzz) [reply resources/entrypoint.sh:5]: Directly from Frank via mail

Comment (szaimen) [reply resources/entrypoint.sh:5]: @blizzz can you ping marketing then that the univention site (https://nextcloud.com/univention) needs updating? (500 is the limit not 100)

**Review by szaimen — APPROVED** [review]

**Pull request merged**